graylog2-web-interface/src/stores/customizations/CustomizationsStore.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow strict
+import Reflux from 'reflux';
+import { qualifyUrl } from 'util/URLUtils';
+import ApiRoutes from 'routing/ApiRoutes';
+import UserNotification from 'util/UserNotification';
+import fetch from 'logic/rest/FetchProvider';
+import ActionsProvider from 'injection/ActionsProvider';
+import { singletonStore } from 'views/logic/singleton';
+
+const CustomizationActions = ActionsProvider.getActions('Customizations');
+
+const urlPrefix = ApiRoutes.ClusterConfigResource.config().url;
+
+type CustomizationsStoreState = {
+  customization: {},
```

Comment:
Should the `{}` be something more specific?

Reply:
Since the customization will be pluggable I do not know what will be in that store state.
So I could add something like:
`[string]: mixed`
but I do not think we will gain much with that.

Reply:
I agree that we will not gain much, but perhaps we should still use `[string]: mixed` for consistency as it seems we use that more often?
I will defer to your judgement here though as my Flow experience is minimal.